x-pack/plugins/security_solution/public/management/pages/index.test.tsx

```diff
@@ -22,15 +23,13 @@ describe('when in the Admistration tab', () => {
 
   it('should display the No Permissions view when Ingest is OFF', async () => {
     (useIngestEnabledCheck as jest.Mock).mockReturnValue({ allEnabled: false });
-    const renderResult = render();
-    const noIngestPermissions = await renderResult.findByTestId('noIngestPermissions');
-    expect(noIngestPermissions).not.toBeNull();
+
```

Comment:
No change in test here, correct? (I'm assuming IDE suggested refactor?)

Reply:
Nope no change in the test and nope no IDE suggestions :). I was just missing the reason to have a variable declaration for cases where it's used only once. I can revert if you think it was better like that.